Resources/Storage/GFAL2_StorageBase.py

```diff
@@ -62,10 +62,14 @@ def __init__( self, storageName, parameters ):
     self.isok = True
 
     # # gfal2 API
-    self.gfal2 = gfal2.creat_context()
+    self.ctx = gfal2.creat_context()
 
     # by default turn off BDII checks
-    self.gfal2.set_opt_boolean( "BDII", "ENABLE", False )
+    self.ctx.set_opt_boolean( "BDII", "ENABLE", False )
+
+    # Avoid caching because of a bug in globus (https://its.cern.ch/jira/browse/DMC-853)
```

Comment:
Maybe add in the comment a "FIXME:" so maybe one day we'll be reminded to remove this line.